docker-compose.yaml

```diff
@@ -53,7 +56,20 @@ services:
       - SANDBOX_PROTOCOL=${SANDBOX_PROTOCOL:-}
       - START_WRITERS=${START_WRITERS:-}
       - CONTRACT_ID=${CONTRACT_ID:-}
-      - CHAIN_CACHE_TYPE=${CHAIN_CACHE_TYPE:-}
+      - CHAIN_CACHE_TYPE=${CHAIN_CACHE_TYPE:-redis}
+      - REDIS_CACHE_URL=${REDIS_CACHE_URL:-redis://redis:6379}
+      - REDIS_CACHE_TTL_SECONDS=${REDIS_CACHE_TTL_SECONDS:-r}
+    depends_on:
+      - redis
+
+  redis:
+    image: redis:latest
+    # TODO: append only is reccomended to prevent losing data
+    # command: redis-server --appendonly yes
```

Comment:
Setting `appendonly` sounds good. We should also limit max memory to something reasonable (2GB seems fine).

Reply:
updated here and allowed an env var should operators have a preference [2dcf902](https://github.com/ar-io/ar-io-node/pull/62/commits/2dcf902c0d9b9583ca72125fd2ff8b86d3063604)